### PR TITLE
Translate OOP to Ukrainian

### PR DESCRIPTION
**Pull request by Herobread**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Oleksii Nazarenko

## Language: 

- Ukrainian

## Terms defined:

- OOP

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -7633,7 +7633,13 @@
       Um paradigma de programação no qual dados (atributos) e funções (métodos)
       são encapsulados em objetos que interagem entre si por meio de
       interfaces bem definidas.
-
+  uk:
+    term: "об'єктно-орієнтоване програмування"
+    acronym: "ООП"
+    def: >
+      Стиль програмування, в якому функції та дані зв'язуються разом в 
+      [об'єкти](#object), які взаємодіють один з одним тільки через
```
Comment (Dod900ls1) [inline glossary.yml:7641]: "взяємодіють між собою" - коротше та як на мене звучить краще якщо читати вголос

File: glossary.yml [hunk 2]
```diff
@@ -7633,7 +7633,13 @@
       Um paradigma de programação no qual dados (atributos) e funções (métodos)
       são encapsulados em objetos que interagem entre si por meio de
       interfaces bem definidas.
-
+  uk:
+    term: "об'єктно-орієнтоване програмування"
+    acronym: "ООП"
+    def: >
+      Стиль програмування, в якому функції та дані зв'язуються разом в
```
Comment (Dod900ls1) [inline glossary.yml:7640]: Змінити "зв'язуються" на "Об'єднуются"?

**Review by Dod900ls1 — CHANGES_REQUESTED** [review]
Some suggestions.

**Comment by Dod900ls1** [comment]
Can you merge those 2 commits together using fixup command in rebase -i?

`git rebase -i HEAD~2`

you'll see smth like

```
pick 18a1f6df2295eba9444a78b8dbcae13a84f8d4ca Translate OOP to Ukrainian
pick abc123 update OOP in Ukrainian
```

you'll need to change it to 

```
pick 18a1f6df2295eba9444a78b8dbcae13a84f8d4ca Translate OOP to Ukrainian
fixup abc123 update OOP in Ukrainian
```
and exit rebase interface

**Comment by Herobread** [comment]
see #705

**Comment by olexandr-konovalov** [comment]
Something went wrong - this got other commits from other places, I see ... There was a way to fix this without a new branch - next time, please get in touch, we can arrange a call and fix

**Comment by olexandr-konovalov** [comment]
P.S. don't have time to investigate what went wrong, but to begin with, the instructions should have started with checking that you're on a right branch...

**Comment by Dod900ls1** [comment]
My bad

**Comment by Herobread** [comment]
ok, will be careful next time

**Pull request closed without merging**